mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/AbstractNioWorker.java

```diff
@@ -629,9 +626,13 @@ public void run() {
                     int interestOps = channel.getInternalInterestOps();
                     interestOps |= SelectionKey.OP_WRITE;
                     channel.setInternalInterestOps(interestOps);
-                    ReadDispatcher readDispatcher = channel instanceof NioSocketChannel
-                            ? new TcpReadDispatcher((NioSocketChannel) channel)
-                            : new UdpReadDispatcher((NioDatagramChannel) channel);
+                    ReadDispatcher readDispatcher = null;
```

Comment:
The indentation was ok. But there were few tabs in there that messed it up in github. I don't know from where I have those tabs from time to time...